calculator_test.go

```diff
@@ -24,31 +24,43 @@ func TestSubtract(t *testing.T) {
 }
 
 type TestCase struct {
-	A    float64
-	B    float64
+	a    float64
+	b    float64
+	f    func(float64, float64) float64
 	want float64
+	name string
 }
 
-func TestMultiply(t *testing.T) {
+func TestCalculator(t *testing.T) {
 	t.Parallel()
 	testCases := []TestCase{
 		{
-			A:    2,
-			B:    2,
+			a:    2,
```

Comment:
For compactness, we could write `a` and `b` on the same line, couldn't we—they're conceptually the same thing (input to the function).